src/main/java/us/kilroyrobotics/subsystems/Elevator.java

```diff
@@ -97,49 +100,56 @@ public Elevator() {
     }
 
     public double getVelocity() {
```

Comment:
Change this from getVelocity to getAppliedOutput.

Reply:
Will do.